### ISSUE TITLE
Circular imports in property.py

### ISSUE THREAD

**Issue by achilleas-k** [G-Node/python-odml#100]
In `property.py` we have an import statement in the body of a method:
https://github.com/G-Node/python-odml/blob/master/odml/property.py#L119

This is done to avoid the circular import property.py → section.py → property.py.
We should find a cleaner way to resolve this.

**Comment by msuzen** [comment]
@achillea Good catch. I had a similar issue on #65, there are `import` within methods because of this.
